lib/Common/CommonPal.h

```diff
@@ -554,3 +567,61 @@ void TryFinally(const TryFunc& tryFunc, const FinallyFunc& finallyFunc)
 
     finallyFunc(hasException);
 }
+
+
+namespace PlatformAgnostic
+{
+    __forceinline unsigned char _BitTestAndSet(LONG *_BitBase, int _BitPos)
+    {
+#if defined(__clang__)
```

Comment:
This builds for x86 too.
But I have probably with x86 with clang in general because of existing inline asm.